lib/dor/models/identifiable.rb

```diff
@@ -185,9 +185,7 @@ def update_tag(old_tag, new_tag)
     def get_related_obj_display_title(related_obj, default_title)
       return default_title unless related_obj
 
-      desc_md_ds = related_obj.datastreams['descMetadata']
-      desc_md_ds_title = desc_md_ds ? desc_md_ds.title_info.main_title.first : nil
-      desc_md_ds_title.present? ? desc_md_ds_title : default_title
+      related_obj.full_title || default_title
```

Comment:
do we need to protect against a blank title here? previously it was using a `.present?` check...